frontend/packages/integration-tests-cypress/tests/crud/secrets/from-yaml.cy.ts

```diff
@@ -0,0 +1,35 @@
+import { checkErrors, testName } from '../../../support';
+import { guidedTour } from '../../../views/guided-tour';
+import { secrets } from '../../../views/secret';
+
+const tlsSecretName = 'tlssecret';
+
+describe('From YAML secrets', () => {
+  before(() => {
+    cy.login();
+    guidedTour.close();
+    cy.createProjectWithCLI(testName);
+    cy.exec(`oc create -f ./fixtures/secret/tlssecret.yaml -n ${testName}`);
+  });
+
+  beforeEach(() => {});
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProjectWithCLI(testName);
+  });
+
+  it('Validate tls secret is editable', () => {
```

Comment:
shall we add this scenario to existing file ‎frontend/packages/integration-tests-cypress/tests/crud/secrets/key-value.cy.ts?